package.json

```
@@ -8,6 +8,19 @@
     "icons"
   ],
   "main": "index.js",
+  "exports": {
+    ".": {
+      "module": "./index.js",
+      "import": "./index.js",
+      "default": "./index.js"
+    },
```

I realize now this should probably just be:

```suggestion
    ".": "./index.js",
```